vignettes/adrs_pcwg3.Rmd

```diff
@@ -0,0 +1,516 @@
+      AVALC == "SD" ~ 3,            # Stable Disease
+      AVALC == "NED" ~ 4,           # No Evidence of Disease
+      AVALC == "Non-CR/Non-PD" ~ 5, # Neither Complete Response nor Progressive Disease
+      AVALC == "NE" ~ 6,            # Not Evaluable
+      AVALC == "PDu" ~ 7,           # Progressive Disease Unconfirmed
+      AVALC == "PD" ~ 8,            # Progressive Disease
+      TRUE ~ NA_real_               # Default for unexpected/missing AVALC values
+    )
+  )
+```
+
+```{r, echo=FALSE}
+dataset_vignette(
+  adrs %>%
+    arrange(!!!get_admiral_option("subject_keys"), PARAMN,),
+  display_vars = exprs(USUBJID,PARCAT1,PARAMCD, PARAM,PARAMN,AVISIT,AVISITN,ADT,AVALC,AVAL)
+)
+```
+
+
+
+### Best Overall Response (BOR) and Confirmed Best Overall Response (CBOR)
+
+BOR represents the Best Overall Responses observed during the study, reflecting valid tumor responses such as Complete Response (CR), Partial Response (PR), Stable Disease (SD), and Progressive Disease (PD). *Progressive Disease Unconfirmed (PDu)* is not included in BOR unless it is confirmed as PD based on the 2+2 flare rule, as per PCWG3 and RECIST 1.1 guidelines.
+
+CBOR, on the other hand, refers to the Confirmed Best Overall Response, requiring sustained responses like CR and PR to meet confirmation criteria (e.g., 28-day confirmation). If *PDu* is not confirmed due to missing follow-up data, CBOR excludes PDu. 
+This ensures CBOR includes only validated and confirmed tumor responses, maintaining compliance with clinical guidelines.
+
+
+
+```{r, eval=TRUE, include=TRUE, message=FALSE}
+
+# Define events forBest Overall Response (BOR)
+bor_cr <- event(
+  description = "Complete Response (CR) for Best Overall Response",
+  dataset_name = "adrs",  
+  condition = AVALC == "CR", 
+  set_values_to = exprs(AVALC = "CR")
+)
+
+bor_pr <- event(
+  description = "Partial Response (PR) for Best Overall Response",
+  dataset_name = "adrs", 
+  condition = AVALC == "PR",
+  set_values_to = exprs(AVALC = "PR")
+)
+
+bor_sd <- event(
+  description = "Stable Disease (SD) for Best Overall Response",
+  dataset_name = "adrs",
+  condition = AVALC == "SD",
+  set_values_to = exprs(AVALC = "SD")
+)
+
+
+bor_pd <- event(
+  description = "Progressive Disease (PD) for Best Overall Response",
+  dataset_name = "adrs",
+  condition = AVALC == "PD",
+  set_values_to = exprs(AVALC = "PD")
+)
+
+bor_ne <- event(
+  description = "Not Evaluable (NE) for Best Overall Response",
+  dataset_name = "adrs",
+  condition = AVALC == "NE",
+  set_values_to = exprs(AVALC = "NE")
+)
+
+
+```
+## Derive Best Overall Response (BOR)
+
+Use the defined events to derive BOR based on the first occurrence in the adrs dataset, prioritizing 
+responses hierarchically (CR > PR > SD > PD > NE).
+
+**Progressive Disease Unconfirmed (PDu)** is excluded as it represents a provisional status and 
+can only be escalated to **Progressive Disease (PD)** if confirmed by the "2+2 flare rule."
+
```

Comment:
```suggestion
```
See comment at the beginning of the section "Best Overall Response (BOR) and Confirmed Best Overall Response (CBOR)".